### ISSUE TITLE
ipython kernel name / environment

### ISSUE THREAD

**Issue by thempel** [markovmodel/pyemma_tutorials#24]
Apparently, @cwehmeyer  and me already had to install the notebooks in two different ways. The problem is that the standard kernel name, even when starting the notebook server from within the environment, does not necessarily point to the right python installation / environment. In my case, it looks up the kernels stored in `~/.local/share/jupyter/kernels/` and takes the kernel named `python3` in from this folder. That might only be an issue for people that use different environments, I don't know. I suggest that we either
- add a note to the readme on how to install the ipython kernel:
```
source activate pyemma_tutorial
python -m ipykernel install --name pyemma_tutorial --user
``` 
and use `pyemma_tutorial` for the kernel name in the notebooks 
or 
- make this implicit by defining the path it in the notebook metadata (don't know if that's possible though).

@marscher any ideas on that?

**Comment by marscher** [comment]
I think we should find a technical solution to avoid people having to set/install the kernel manually. There are two local configuration files, which can be placed into the notebook folder:
- jupyter_config.py
- jupyter_config.json

Maybe one can configure the kernel there. I would also suggest to set the kernel name to something specific like "PyEMMA-Tutorials-Py3" to ensure people are using the right environment.

**Comment by thempel** [comment]
I agree, we should make it as easy to run as possible. Are you sure that it is possible to set these jupyter config files for a single directory?

Since we already have two commands for people to install the conda environment in our readme, maybe it would be the easiest to add a third one (`python -m ipykernel install --name PyEMMA-Tutorials-Py3 --user`) and to put everything into an `.sh` script. This would make our environment available in the dropdown menu in the notebooks for the user. If we set the default kernel name in the notebooks to the same name, this should also work out of the box.

**Comment by cwehmeyer** [comment]
What's the status here?

**Comment by marscher** [comment]
fixed in #34
